### PR TITLE
[de] update translation

### PR DESCRIPTION
**Pull request by sotten**
update german [de] translation

### CODE REVIEW

File: excludes/de.php [hunk 1]
```diff
@@ -12,8 +12,108 @@
...
+    'Botswana',
+    'Burkina Faso',
+    'Burundi',
+    'Chile',
+    'China',
     'Code',
+    'Costa Rica',
+    'Curaçao',
     'Dashboard',
+    'Details',
```
Comment (andrey-helldar) [inline excludes/de.php:39]: `Details` > `Einzelheiten`?

Don't translate this word for sure?
Google Translate says there is a translation.

Comment (sotten) [reply excludes/de.php:39]: Yes I am sure.
Details can be translated with details.
Check "More details" in Google Translate. It translates it with "Mehr Details".
In German we often use the word "Details" for "show me more information about an object"

File: excludes/de.php [hunk 2]
```diff
@@ -12,8 +12,108 @@
...
+    'Haiti',
+    'Honduras',
+    'ID',
+    'Israel',
+    'Japan',
+    'Jersey',
+    'Kiribati',
+    'Kosovo',
+    'Kuwait',
+    'Lens',
```
Comment (sotten) [inline excludes/de.php:65]: Is "Lens" a geographical place/area/location?
In what context is this term used?  Where is it used? Laravel Nova?

"Lens" in German is "Objektiv" or "Linse"

File: excludes/de.php (hunk 1 above)
Comment (andrey-helldar) [reply excludes/de.php:39]: Got it. Just asked :)

I know that Google Translate doesn't translate exactly, but this word has a confirmation badge from people.

I think the confirmation is out of context.

image

File: excludes/de.php (hunk 2 above)
Comment (andrey-helldar) [reply excludes/de.php:65]: @masterix21, can you help with this question?

Comment (andrey-helldar) [reply excludes/de.php:65]: In russian "Lens" is

image

too

Comment (andrey-helldar) [reply excludes/de.php]: @sotten, see: https://nova.laravel.com/docs/3.0/lenses/defining-lenses.html

Comment (andrey-helldar) [reply excludes/de.php]: It seems to me that they called scopes and repositories with the word "lenses"...

Comment (sotten) [reply excludes/de.php]: "Lens" seems to come from Laravel Nova and has nothing to do with geography  ;)
In the Nova documentation this occurs more often.  I think it has something to do with filters or something like that.  I removed Lens from the exclusion list. Maybe it's a Nova proper name too.

Comment (andrey-helldar) [reply excludes/de.php]: Now I looked, I [translated](https://github.com/Laravel-Lang/lang/blob/master/locales/ru/ru.json#L283) this word into Russian - "Линза" (en: Lens, de: Linse).

Comment (caouecs) [reply excludes/de.php]: the exclusion list is not just for geography

**Comment by caouecs** [comment]
Danke

File: excludes/de.php (hunk 2 above)
Comment (andrey-helldar) [reply excludes/de.php]: Yes, but we couldn't figure out in what context Laravel Nova uses the word "Lens" :)

**Pull request merged**